src/Parsers.jl

```diff
@@ -18,10 +18,15 @@ function __init__()
     end
     Threads.resize_nthreads!(STRINGBUFFERS)
     Threads.resize_nthreads!(ONES)
+    foreach(x->MPZ.init!(x), ONES)
```

Comment:
You can just make this `foreach(MPZ.init!, ONES)`, likewise for all of the others.